modules/passkey/README.md

```diff
@@ -2,9 +2,29 @@
 
 This package contains a passkey signature verifier, that can be used as an owner for a Safe, compatible with versions 1.3.0+.
 
-## SafeWebAuthnSignerProxy
+## Contracts overview
 
-Use of `SafeWebAuthnSignerProxy` provides gas savings compared to the complete bytecode contract for each signer creation. The `SafeWebAuthnSignerProxy` contract is a proxy contract that forwards calls to the `SafeWebAuthnSignerSingleton` contract which is a singleton contract. Both `SafeWebAuthnSignerProxy` and `SafeWebAuthnSignerSingleton` use no storage slots to avoid storage access violations defined in ERC-4337. The details on gas savings can be found in [this PR](https://github.com/safe-global/safe-modules/pull/370).
+Safe account being standard agnostic, new user flows such as custom signature verification logic can be added/removed as and when required. By leveraging this flexibility to support customizing Safe account, Passkeys-based execution flow can be enabled on a Safe. The contracts in this package use [ERC-1271](https://github.com/frangio/eip712-wrapper-for-eip1271) standard and WebAuthn standard to allow signature verification on SECP256K1 curve. The contracts in this package are designed in such a way that they can be used with precompiles for signature verification in the supported networks or use any verifier contract as a fallback mechanism. In their current state, the contracts are tested with [Fresh Crypto Lib (FCL)](https://github.com/rdubois-crypto/FreshCryptoLib).
```

Comment:
Please link the EIP directly instead of a 3rd party repository

Reply:
ooh.. I copied link from another tab. Fixed it here https://github.com/safe-global/safe-modules/pull/394/commits/093a298e87950c363b0664f95d0aea434446760b